Scripts/build-derived-assets.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+#  copy-asset-overrides.sh
+#  Loop
+#
+#  Copyright © 2019 LoopKit Authors. All rights reserved.
+
+
+echo "Building DerivedAssets.xcassets"
+
+output="${BUILT_PRODUCTS_DIR}/DerivedAssets.xcassets"
+cp -a "${PROJECT_DIR}/Loop/DefaultAssets.xcassets/." "$output"
+
+override="${PROJECT_DIR}/../AssetOverrides.xcassets/."
```

Comment:
The purpose of this script is an attempt to support both styles of builds. It checks if the asset overrides exist.  And otherwise doesn't attempt to overlay them.  Unfortunately, we still have the warning popup.  I'm not sure how to get around that, other than removing the assets from the build entirely, and copying them into the target with a script.

Reply:
:+1: